frame/vesting/README.md

```diff
@@ -15,6 +15,8 @@ equivalent to the amount remaining to be vested. This is done through a dispatch
 either `vest` (in typical case where the sender is calling on their own behalf) or `vest_other`
 in case the sender is calling on another account's behalf.
 
+A user can have multiple vesting schedules, which are capped by an upper bound. With an upper bound
+>= 2, users can merge 2 schedules.
```

Comment:
Leading a markdown line with `>` might confuse the parser; in markdown, that means "blockquote". Breaking the line elsewhere is probably a good idea.